dvc/exceptions.py

```diff
@@ -310,3 +310,11 @@ def __init__(self, path, repo):
             " neither as an output nor a git-handled file."
         )
         super().__init__(msg.format(path, repo))
+
+
+class RemoteCacheRequiredError(DvcException):
+    def __init__(self, scheme):
+        super().__init__(
+            "Remote cache for '{}' remote is required to perform this "
+            "operation".format(scheme)
```

Comment:
Could we add something helpful or add a link to docs?

Reply:
Also "this operation" is not very clear. Maybe we could include path too to be more precise about what path has caused this exception?